### PR TITLE
Remove `mc admin bucket remote bandwidth` command and introduce

### PR DESCRIPTION
**Pull request by poornas**
bandwidth as part of `mc replicate status` to show bandwidth limit and usage per target

## Motivation and Context
Move all replication relevant metrics into `mc replicate status`

## How to test this PR?
use https://github.com/minio/minio/pull/16360 for server side changes
## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fmc%2fpull%2fNNNNN)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
PTAL @poornas conflicts.

**Comment by harshavardhana** [comment]
PTAL @vadmeste

File: cmd/replicate-status.go [hunk 1]
```diff
@@ -157,21 +158,40 @@ func (s replicateStatusMessage) String() string {
...
 			r = console.Colorize("THeader", newPrettyTable(" | ",
-				Field{"Arn", 120},
-			).buildRow("  "+hdrDet))
+				Field{"Arn", 100},
+			).buildRow("  "+"ARN: "+hdrDet))
 			rows += r
 			rows += "\n"
+			bwStat, ok := s.ReplicationStatus.Stats[arn]
+			if ok && bwStat.BandWidthLimitInBytesPerSecond > 0 {
+				limit := humanize.Bytes(uint64(bwStat.BandWidthLimitInBytesPerSecond) * 8)
```
Comment (vadmeste) [inline cmd/replicate-status.go:178]: Why is there "x 8" here ?

**Review by vadmeste — COMMENTED** [review]
One comment

File: cmd/replicate-status.go (hunk 1 above)
Comment (harshavardhana) [reply cmd/replicate-status.go:178]: Because the data sent is in bits @vadmeste

Comment (poornas) [reply cmd/replicate-status.go:178]: the server stores bandwidth limit in bytes/sec whereas the mc UI takes bandwidth-limit flag in bits/sec - hence the conversion while displaying `mc replicate status`

Comment (vadmeste) [reply cmd/replicate-status.go:178]: > the server stores bandwidth limit in bytes/sec whereas the mc UI takes bandwidth-limit flag in bits/sec - hence the conversion while displaying mc replicate status

understood, but also you are passing bits to humanize.Bytes(), I don't think it is working correctly.

Comment (poornas) [reply cmd/replicate-status.go]: updated PR to display as Bytes/sec instead

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**